xcconfig/xcconfig.go

```diff
@@ -2,34 +2,52 @@ package xcconfig
 
 import (
 	"fmt"
+	"path/filepath"
+	"strings"
+
 	"github.com/bitrise-io/go-utils/v2/fileutil"
 	"github.com/bitrise-io/go-utils/v2/pathutil"
-	"path/filepath"
 )
 
 // Writer ...
 type Writer interface {
-	Write(content string) (string, error)
+	Write(input string) (string, error)
```

Comment:
This would be the perfect time to add a few lines of documentation to this interface and method. At the moment it's not clear what the `input` string can contain (file content or path) and what happens in each case.